UserInterface/MenuItemDemos/MenuItemDemos/MenuItemDemos/MenuItemXamlMvvmPage.xaml

```diff
@@ -0,0 +1,43 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:d="http://xamarin.com/schemas/2014/forms/design"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:viewmodels="clr-namespace:MenuItemDemos.ViewModels"
+             mc:Ignorable="d"
+             x:Class="MenuItemDemos.MenuItemXamlMvvmPage"
+             x:Name="contentPage"
+             Padding="10"
+             Title="MenuItem XAML MVVM Demo">
+    
+    <ContentPage.BindingContext>
+        <viewmodels:ListPageViewModel />
+    </ContentPage.BindingContext>
+
+    <StackLayout>
+        <Label Text="{Binding Message}"
+               TextColor="Red"
+               HorizontalOptions="CenterAndExpand"
+               VerticalOptions="Start" />
+        <ListView ItemsSource="{Binding Items}"
+                  HorizontalOptions="CenterAndExpand"
+                  VerticalOptions="CenterAndExpand">
+            <ListView.ItemTemplate>
+                <DataTemplate>
+                    <ViewCell>
+                        <ViewCell.ContextActions>
+                            <MenuItem Text="Edit"
+                                      IconImageSource="icon.png"
```

Comment:
Wondering what this is for? I guess it's demoing that you can set this property. But it's quite non-intuitive that the Xamarin logo is an edit button on Android.